src/hooks/useWalletConnection.ts

```diff
@@ -138,6 +140,8 @@ export const useWalletConnection = () => {
               walletType: cosmosWalletType,
             });
           }
+        } else if (walletConnection.type === WalletConnectionType.TestWallet) {
+          saveEvmAddress(STRING_KEYS.TEST_WALLET as EvmAddress);
```

Comment:
I don't think we need to have the dummy address be localized, so probably don't need to use `STRING_KEYS.TEST_WALLET`. Just a constant for `DUMMY_EVM_ADDRESS` should be good.